### DISCUSSION TITLE
Can APatch be put to the Android Emulator?

### DISCUSSION THREAD

**Discussion by llv22** [bmax121/APatch#520] [Q&A]
As title, I want to know if APath can be applied to Android Emulator?

**Comment by uontabc** [comment]
No,apatch isn't in Android Emulator:),it have many bugs,If you don't mind, you can try

**Comment by Faizan-26** [comment]
Same Question after 2 years?
Can u apply Apatch to any android emulator even outside of android studio emulators like LDPlayer, BlueStack etc
